airbnb-app/src/component/search-bar/SearchBar.jsx

```diff
@@ -1,14 +1,52 @@
+import styled from 'styled-components';
 import SEARCH_INPUT_TEXT from '../../constants/searchBarText';
 import SearchInput from './SearchInput';
+import { useState, Fragment } from 'react';
+
+const searchInputText = Object.entries(SEARCH_INPUT_TEXT);
 
 function SearchBar() {
+  const [currentInput, setCurrentInput] = useState(null);
+  const isLastElement = index => index === searchInputText.length - 1;
+  const isCurrentInput = label => label === currentInput;
+  const isFocus = currentInput !== null;
+
+  const handleBlur = () => {
+    setCurrentInput(null);
+  };
+
   return (
-    <>
-      {Object.entries(SEARCH_INPUT_TEXT).map(([key, { label, placeholder }]) => (
-        <SearchInput key={key} label={label} placeholder={placeholder} />
-      ))}
-    </>
+    <Form method="POST">
+      <SearchMenu bgColor={isFocus ? 'grey6' : 'white'} tabIndex="0" onBlur={handleBlur}>
+        {searchInputText.map(([key, { label, placeholder }], index) => (
+          <Fragment key={key}>
+            <SearchInput
+              label={label}
+              placeholder={placeholder}
+              isLastElement={isLastElement(index) ? true : false}
+              isCurrentInput={isCurrentInput(label) ? true : false}
```

Comment:
```suggestion
              isCurrentInput={isCurrentInput(label)}
```

boolean을 반환하는 함수일 경우에는 또 true/false로 변환해 주지 않아도 됩니다!